feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -176,51 +174,64 @@ func configForwardingPolicy() *oc.NetworkInstance_PolicyForwarding {
 	ni := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
 	ipv4Address := "0.0.0.0/0"
 	ipv6Address := "::/0"
-	ipv6dest := "2001:db8::1/128"
+	icmp6 := 58
 
 	// Match policy
 	policyFwding := ni.GetOrCreatePolicyForwarding()
 
 	fwdPolicy1 := policyFwding.GetOrCreatePolicy("match-ipv4")
+	fwdPolicy1.SetType(oc.Policy_Type_PBR_POLICY)
 	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
 	fwdPolicy1.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
-	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6dest)
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv6().Protocol = oc.UnionUint8(icmp6)
 	fwdPolicy1.GetOrCreateRule(2).GetOrCreateAction().Discard = ygot.Bool(false)
 	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv4().DestinationAddress = ygot.String(ipv4Address)
 	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv6().DestinationAddress = ygot.String(ipv6Address)
 	fwdPolicy1.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(true)
 
 	fwdPolicy2 := policyFwding.GetOrCreatePolicy("match-ipip")
+	fwdPolicy2.SetType(oc.Policy_Type_PBR_POLICY)
```

Comment:
requirement in HLD is for vrf selection.  oc.Policy_Type_VRF_SELECTION_POLICY  is an appropriate policy type for vrf selection.